app/assets/javascripts/discourse/app/controllers/sidebar-section-form.js

```diff
@@ -200,7 +205,6 @@ export default Controller.extend(ModalFunctionality, {
             icon: link.icon,
             name: link.name,
             value: link.path,
-            external: link.external,
```

Comment:
This is leftover from previous solution. External is calculated by backend and this parameter is not required.

https://github.com/discourse/discourse/blob/main/app/models/sidebar_url.rb#L30